### DISCUSSION TITLE
Cannot change time format

### DISCUSSION THREAD

**Discussion by mjm455** [poblabs/weewx-belchertown#949] [Q&A] [answered]
I'm using Belchertown v1.3.1 and it is running great.  I'm not a fan of the 24h time format and it is not changing when I edit skin.conf. 
 I really want it to look like the [authors page for Belchertown](https://belchertownweather.com).  Here is what that section of my file looks like:

    [[TimeFormats]]
                 day        = %I:%M
                 week       = %I:%M on %A
                 month      = %d-%b-%Y %I:%M
                 year       = %d-%b-%Y %I:%M
                 rainyear   = %d-%b-%Y %I:%M
                 current    = %d-%b-%Y %I:%M
                 ephem_day  = %I:%M
                 ephem_year = %d-%b-%Y %I:%M

What am I doing wrong?  I've looked for other discussions or closed issues but couldn't find much.

**Comment by ervu** [comment]
Hi change in weewx.conf the "belchertown_locale" options from auto to "en_GB.UTF-8" and your time will switch to 24h format.
To avoid errors in the weewx logs files you local os need to have the locals package installed.

apt-get install locales
dpkg-reconfigure locales option 87 option 3
https://serverfault.com/questions/362903/how-do-you-set-a-locale-non-interactively-on-debian-ubuntu

**Comment by mjm455** [comment]
Thanks for the reply.  It is already set to a 24h format.  How do I get it to 12h?

**Comment by ervu** [comment, marked as answer]
aha... misread your post.... 
Have you tried this setting ?

belchertown_locale = "en_US.UTF-8" ##  change for 12 AM/PM h date time format

this is a list of all the locales
https://docs.oracle.com/cd/E23824_01/html/E26033/glset.html
